types/reactstrap/lib/Popover.d.ts

```diff
@@ -7,6 +7,7 @@ export type PopoverProps<T = {}> = React.HTMLAttributes<HTMLElement> & {
   toggle?: () => void;
   target: string | HTMLElement;
   container?: string | HTMLElement;
+  boundariesElement?: Popper.Boundary | Element;
```

Comment:
That's a 6.4 update, you need to increase the version number in the index.d.ts file. https://github.com/reactstrap/reactstrap/releases/tag/6.4.0